README.md

```diff
@@ -1,4 +1,4 @@
-# Magento Developer Documentation
+Â# Magento Developer Documentation
```

Comment:
Odd little A seems to made it in this line?

Reply:
How'd that get there?! Fixed it.